web/admin/src/ContentDetailsEditor.js

```diff
@@ -281,14 +281,44 @@ export default class ContentDetailsEditor extends PureComponent {
         return (
           <div className="content-editor__box" key={`${content.type}Fields`}>
             <h2 className="contentTitle">{t('survey')}</h2>
-            <SelectEditor
-              size={6}
+            {!content.surveyURL && (
+              <SelectEditor
+                size={6}
+                content={content}
+                prop="surveyId"
+                title="Survey"
+                onUpdate={this.onUpdateSurvey}
+                options={surveys}
+              />
+            )}
+            <TextEditor
               content={content}
-              prop="surveyId"
-              title="Survey"
-              onUpdate={this.onUpdateSurvey}
-              options={surveys}
+              prop="surveyURL"
+              title="Optional: Use an Advanced Survey by entering its URL"
+              placeholder="dd://extensions/surveys?surveyID=-J123Sna323cn"
```

Comment:
Different people have different opinions on how to case 'ID' (since it's an abbreviation rather than an acronum).  Let's use `surveyId` and keep casing consistent with how we case `Id` elsewhere.